src/fides/api/models/privacy_experience.py

```diff
@@ -219,260 +561,41 @@ class PrivacyExperience(Base):
-            # but this is subject to change.
-            return True
-
-        if self.component != ComponentType.overlay:
-            return False
-
-        if self.experience_config:
-            if self.experience_config.banner_enabled == BannerEnabled.always_disabled:
-                return False
-
-            if self.experience_config.banner_enabled == BannerEnabled.always_enabled:
-                return True
-
-        privacy_notice_query = get_privacy_notices_by_region_and_component(
-            db, [self.region.value], self.component  # type: ignore[arg-type, attr-defined]
-        )
-        if show_disabled is False:
-            privacy_notice_query = privacy_notice_query.filter(
-                PrivacyNotice.disabled.is_(False)
-            )
-
-        return bool(
-            privacy_notice_query.filter(
-                PrivacyNotice.consent_mechanism.in_(BANNER_CONSENT_MECHANISMS)
-            ).count()
-        )
-
     @staticmethod
-    def create_default_experience_for_region(
-        db: Session, region: PrivacyNoticeRegion, component: ComponentType
-    ) -> PrivacyExperience:
-        """Creates an Experience for a given Component Type and Region and links
-        to default copy (ExperienceConfig)"""
-        experience_data: Dict = {
-            "region": region,
-            "component": component,
-        }
-        default_config: Optional[
-            PrivacyExperienceConfig
-        ] = PrivacyExperienceConfig.get_default_config(db, component)
-
-        if default_config:
-            experience_data["experience_config_id"] = default_config.id
-
-        return PrivacyExperience.create(
-            db=db,
-            data=experience_data,
-        )
-
-    @staticmethod
-    def get_experience_by_region_and_component(
+    def get_experiences_by_region_and_component(
         db: Session, region: str, component: ComponentType
-    ) -> Optional[PrivacyExperience]:
-        """Load an experience for a given region and component type"""
+    ) -> Query:
+        """Utility method to load experiences for a given region and component type"""
         return (
             db.query(PrivacyExperience)
+            .join(
+                PrivacyExperienceConfig,
+                PrivacyExperienceConfig.id == PrivacyExperience.experience_config_id,
+            )
             .filter(
                 PrivacyExperience.region == region,
-                PrivacyExperience.component == component,
-            )
-            .first()
-        )
-
-    @staticmethod
-    def get_overlay_and_privacy_center_experience_by_region(
-        db: Session, region: str
-    ) -> Tuple[Optional[PrivacyExperience], Optional[PrivacyExperience]]:
-        """Load both the overlay and privacy center experience for a given region
-
-        TCF overlays are not returned here.  This method is used in building experiences when Notices
-        are created, which is not applicable for TCF.
-        """
```

Comment:
Logic not really relevant anymore.  There are more and different experiences than this now, and the reason for this method doesn't exist anymore either.